### ISSUE TITLE
Wheels with LLVM backend

### ISSUE THREAD

**Issue by whitequark** [wasmerio/wasmer-python#192]
### Motivation

The Wasmer LLVM backend provides support for a lot more platforms than Cranelift. At the moment if I am depending on Wasmer I would need to ask users on platforms other than x86_64/AArch64 to build Wasmer manually, which is a significant hurdle.

### Proposed solution

It would be nice if the Wasmer Python extension had wheels published with the LLVM backend enabled.

**Comment by Hywan** [comment]
Asked if it is possible to rename a wheel name, https://github.com/PyO3/maturin/issues/313.

**Comment by Hywan** [comment]
While waiting on finding a solution for that, I would like to underline that it's already possible to generate a wheel with a specific backend/compiler, see https://github.com/wasmerio/python-ext-wasm#use-a-particular-wasmer-compiler-backend to learn more.

For instance:

```sh
$ just --set build_features backend-llvm build
```

**Comment by whitequark** [comment]
Yeah, I know that. Unfortunately it's not useful for my purpose, which is distributing packages on PyPI that depend on a WASI runtime also on PyPI.

**Comment by Hywan** [comment]
My main problem is that I don't want to publish 4 packages: `wasmer` (with a default compiler), `wasmer-singlepass`, `wasmer-cranelift` and `wasmer-llvm`, each with the same API. It's confusing. I'm thinking of a better solution, I'll need a couple of weeks.

**Comment by whitequark** [comment]
Sounds good.

**Comment by imachug** [comment]
What solution have you come up with?

**Comment by Hywan** [comment]
Closing in favor of https://github.com/wasmerio/wasmer-python/issues/262, see this issue to understand the progress please :-).
